modules.d/01systemd-udevd/module-setup.sh

```diff
@@ -16,9 +16,18 @@ check() {
 
 # Module dependency requirements.
 depends() {
-    # This module has external dependency on other module(s).
-    echo udev-rules systemd systemd-sysctl systemd-modules-load
-    # Return 0 to include the dependent module(s) in the initramfs.
+    local deps
+    deps="udev-rules systemd"
+
+    # install optional dependencies unless there are omitted
```

Comment:
s/there/they

Reply:
fixed. Thanks !

Reply:
I applied your patches (minus the third patch modifying the test) to the latest git version and built a ramdisk against systemd[-kmod]. It builds and boots without issue, the boot log looks good.

I don't feel qualified to review the patches, really. I can follow their intent and the logic looks right, fwiw.